### ISSUE TITLE
How to list the GUS (Global Underscore)

### ISSUE THREAD

**Issue by davide-casiraghi** [davide-casiraghi/ci-global-calendar#101]
The Global Underscore (GUS) is an event that involve involves 1000+ dancers! 
There will be many individual Global Underscore (GUS) events around the world, anyway Patrick Crowley feel the need about giving more space to a description of the main event in the website. 

**Describe the solution you'd like**
Maybe a banner or a dedicated page with some more details about the underscore.

Issue reported by Patrick Crowley

**Comment by patrickcrowley101** [comment]
Yes, because it is global event, the "venue" is "EARTH", which is not on the list! Also one has to list a city and country. So the technology does not allow this event to be listed. The importance of a listing of the overall event of GLOBAL UNDERSCORE is so that people can register a site for others to dance at.

**Comment by davide-casiraghi** [comment]
Personally I'm against adding Earth as country or venue, because I think it can create many kind of problems.
People can start adding earth to give more visibility to their events i think is one.
So I think is a better idea to create a dedicated page and maybe use the popup we are using for the donation request to drive people to the page.

**Comment by patrickcrowley101** [comment]
Hey Davide,

Yes an earth listing could bring problems…

A pop up would give a lot of attention to it!

Thanks for thinking about it.

:) Patrick



On Apr 9, 2019, at 3:57 AM, Davide Casiraghi <notifications@github.com> wrote:

Personally I'm against adding Earth as country or venue, because I think it can create many kind of problems.
People can start adding earth to give more visibility to their events i think is one.
So I think is a better idea to create a dedicated page and maybe use the popup we are using for the donation request to drive people to the page.

**Comment by stawastawa** [comment]
popup is decent. 

I think all GUS events should be encouraged to list on the CIGC.  Having them somehow show up for any location would also be good, so no matter where a user is searching the global underscore will show up.  What I mean is having a general GUS event listing, which links to the GUS website and encourages users to setup a local GUS event. 
perhaps a "if no GUS within 100kilometers, show: 'host a GUS?' event. Else, show: local GUS events within 100km."

Having all GUS events link to a Regional page using Iframe would also be nice.  This would be more about making it easy for the GUS website to list all known GUS events.

**Comment by patrickcrowley101** [comment]
We are about to launch the new GUS website. We will encourage facilitators to post on CIGC. Let me know what you decide about a general GUS listing.

Thanks!
Patrick

On Apr 19, 2019, at 1:25 PM, stawastawa <notifications@github.com> wrote:

popup is decent.

I think all GUS events should be encouraged to list on the CIGC. Having them somehow show up for any location would also be good, so no matter where a user is searching the global underscore will show up. What I mean is having a general GUS event listing, which links to the GUS website and encourages users to setup a local GUS event.
perhaps a "if no GUS within 100kilometers, show: 'host a GUS?' event. Else, show: local GUS events within 100km."

Having all GUS events link to a Regional page using Iframe would also be nice. This would be more about making it easy for the GUS website to list all known GUS events.

**Comment by davide-casiraghi** [comment]
When will it be the GUS this year?

**Comment by patrickcrowley101** [comment]
Global Underscore (GUS) 2019
June 22, 2019, 10:00 am - 2:00 pm EST (UTC-04:00)
GUS@globalunderscore.com
www.globalunderscore.com <http://globalunderscore.blogspot.com/>
facebook <https://www.facebook.com/GlobalUnderscore>
Global Underscore Coordinators:
Nancy Hughes, Patrick Crowley
with Nancy Stark Smith
Brandin Steffensen, adviser

On May 1, 2019, at 6:12 PM, Davide Casiraghi <notifications@github.com> wrote:

When will it be the GUS this year?

**Comment by davide-casiraghi** [comment]
Patrick, 
I will address this topic with the calendar team and let you know.
Are the moment after the rollout we have a huge amount of topics open.
To address all of them we would require 2 people working full time. 
I’m spending all my energies at the moment working daily to make all the basic calendar functionalities a separate package developed following a TDD approach.
I’m at a very good stage, after finishing that I will go trough all the bug fixes open in the github issues, then trough the issues of phase 1.
And then trough the issues that the team consider as priorities.
Once finished it will be easier to maintain and improve, alone or for a pool of developers.
